comfyui_utils/comfy.py

```diff
@@ -138,18 +179,24 @@ async def submit(self, prompt: StrDict, callbacks: Callbacks):
         async with aiohttp.ClientSession() as session:
             # Enqueue and get prompt ID.
             async with session.post(f"http://{self.address}/prompt", data=init_data) as resp:
-                response_json = await resp.json()
-                logging.info(response_json)
-                if "error" in response_json:
-                    if "node_errors" not in response_json:
-                        raise ValueError(response_json["error"]["message"])
-                    errors = []
-                    for node_id, data in response_json["node_errors"].items():
-                        for node_error in data["errors"]:
-                            errors.append(f"Node {node_id}, {node_error['details']}: {node_error['message']}")
-                    raise ValueError("\n" + "\n".join(errors))
-
-                prompt_id = response_json['prompt_id']
+                try:
+                    response_json = await resp.json()
+                    logging.info(response_json)
+                    if "error" in response_json:
+                        if "node_errors" not in response_json:
+                            raise ValueError(response_json["error"]["message"])
+                        errors = []
+                        for node_id, data in response_json["node_errors"].items():
+                            for node_error in data["errors"]:
+                                errors.append(f"Node {node_id}, {node_error['details']}: {node_error['message']}")
+                        raise ValueError("\n" + "\n".join(errors))
+
+                    prompt_id = response_json['prompt_id']
+                except aiohttp.client_exceptions.ContentTypeError as e:
```

Comment:
can the except be moved up to only wrap the `await resp.json()` so any other errors bubble up normally?